File: WebApp/WebApp/Models/BusStopsOnLine.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class BusStopsOnLine
    {
        [Key]
        public Guid Id { get; set; }
        [Required]
        public Guid BusStopId { get; set; }
        public BusStop BusStop { get; set; }
        [Required]
        public string LineId { get; set; }
        public Line Line { get; set; }

    }
}

File: angular-app/src/models/bus-stops-on-line.ts
import { Guid } from "guid-typescript";
import { BusStop } from './bus-stop';
import { Line } from './line';

export class BusStopsOnLine {
    public Id: Guid;
    public BusStopId: Guid;
    public BusStop: BusStop;
    public LineId: string;
    public Line: Line;
}